code/modules/mob/living/carbon/xenomorph/XenoProcs.dm

```diff
@@ -145,6 +145,9 @@
 	if(frenzy_aura)
 		. -= (frenzy_aura * 0.05)
 
+	if(rage)
```

Comment:
You can put this into `/mob/living/carbon/Xenomorph/Ravager/movement_delay()` and avoid ravager vars for non-Ravas, as well as redundant checks.
Don't forget to add a `. = ..()` at the beginning.